includes/class-newspack-popups-model.php

```diff
@@ -448,6 +450,7 @@ function( $value, $key ) {
 			$filtered_options,
 			[
 				'background_color'               => '#FFFFFF',
+				'close_button_background_color'  => '',
```

Comment:
Instead of an empty colour, I'd suggest using `#00000000` is basically black at 0% opacity.

Reply:
Done throughout!